gradle/libs.versions.toml

```diff
@@ -12,6 +12,8 @@ spring-boot = "2.7.2"
 spring-security = "5.7.2"
 spring-cloud = "3.1.3"
 spring-cloud-kubernetes = "2.1.3"
+spring-kafka = "2.8.7"
```

Comment:
we use `spring-boot-dependencies` and it contains version for kafka and spring-kafka

Reply:
fixed